tests/integration/test_outcome.py

```diff
@@ -1074,7 +1077,7 @@ def test_graceful_shutdown(relay, mini_sentry):
         "key_id": 123,
         "outcome": 1,
         "reason": "Sampled:1",
-        "category": 2,
+        "category": 9,
```

Comment:
datacategory changed from `Transaction` to `TransactionIndexed`

is this a problem?

Reply:
It actually makes sense to report `Indexed` for sampled transactions, but I don't understand why this changed now. Could you please try removing all the `del`s as suggested above and then see what happens?

Reply:
It makes sense: We pick either category based on the presence of config:

https://github.com/getsentry/relay/blob/7b34191316429a2061ed83b128fd3f35ea45d30e/relay-server/src/utils/rate_limits.rs#L460-L463